### ISSUE TITLE
Removing name/gender/age/etc from application review view

### ISSUE THREAD

**Issue by pholm** [hackjunction/hackplatform#61]
To support equality and truly focus on the skills the applicants present, the application review page should not show name, gender, age and such identifiers for the person who rates the application. 

This is just an idea, but could be a great concrete step towards more equal hackathon community. Any opinions?

**Comment by lappalj4** [comment]
I think it's a good idea. To do this we would probably need to create a separate reviewing-page on the event admin which would:

- get you the chronologically first unreviewed application
- show only a subset of the users details
- allow you to give a rating to them

And then we would additionally have the current table of participants which allows you to see and edit all details of a participant.

**Comment by lappalj4** [comment]
It would also get rid of the need to "assign" people to yourself to avoid duplicate ratings

**Comment by perssonsimon1** [comment]
If no one is working on this issue, I would like to give it a try. Which user details are preferred to be shown on such a page?

**Comment by lappalj4** [comment]
@perssonsimon1 awesome! I'm currently working on a bit of a rework of the event admin view in general, so I think it would be a good idea to wait on that before beginning on this. In the meantime (if you haven't already), see if you can [set up the development environment](https://github.com/hackjunction/hackplatform/blob/dev/DEVELOPMENT.md), set up a test event and get familiar with how the admin panel currently works in general.

I'll let you know once I'm done with the rework 😎

**Comment by perssonsimon1** [comment]
Ok, the dev environment is already up and running. The setup script did almost everything 👌. Just wondering about the ‘:’ prefix of some folders, is there a reason for this? I’m doing the contribution as part of a cs course, meaning I need to have something before Friday. Is there another issue I can get started on while waiting for the rework? 😀

**Comment by lappalj4** [comment]
If you're still interested, you could take a look at #31 (which the backend for has already been implemented and merged in `dev`) and the part regarding allowing users to delete their own profile: 

> So, to make this happen we should create a new tab under the /account page, i.e. "Privacy". Under that tab there will be various privacy-related settings, one of which is this "Forget me" functionality.

> Required features (frontend):

> - A big red dangerous-looking button to initiate the forgetting of oneself
> - Some sort of double-verification, e.g. a popup which asks the user to write "Yes, I'm sure" or something similar.
> - Once the deletion is done, log the user out. After this the user will no longer be able to log in on that account.

**Comment by perssonsimon1** [comment]
Unfortunately we had to pick another project to work on for the course. I can take on the issue myself though, but it would have to be next week then. I’ll get back to you when I get started.

**Comment by Earlo** [comment]
Would this mean that cover letters, CV:s, portfolios, or profiles on external platforms (github etc) couldn't be taken into account in reviewing process?

**Comment by lappalj4** [comment]
That's a good point, in the end it would be impossible to rate someone without revealing their name/gender etc. via these sorts of external profiles. And certainly those are some of the most important things to consider in the reviewing process.

I think if we implement this sort of separate reviewing view, which I think would be useful in any case, it would be trivial to hide these sorts of details anyway so we should probably do it. For some events, the registration form won't include fields like LinkedIn or CV which would reveal such details (nor are they required for most other events), and for example someones GitHub profile doesn't necessarily reveal their gender/nationality. 

So yeah, it's definitely going to be impossible to review some (most?) applications without coming across those details but hiding those fields in the reviewing view within the app would still be a step in the right direction to support equality in some cases.

**Comment by Earlo** [comment]
One option would be to have two step reviewing process, one where external info such as CV and LinkedIn are reviewed, and second seperate where the Junction App profile is reviewed, the complete rating would be a combination of two.

Though, there might be people who neglect their Junction profile since they think that "I already have that info in my CV". But by communicating that we could encourage people to put effort into their profiles.

**Comment by Earlo** [comment]
Suggestion: 

To make this into an feature, change it as ability for organizer to select which fields show up when reviewing participants
